lib/docker.js

```diff
@@ -51,7 +51,8 @@ function runImage(imageId, cwd, command, mount) {
   var mountDir = crossPlatformCwd(cwd);
   var mountComponent = mount ? `-v "${mountDir}:/app/src"` : '';
   var envArgComponent = directory.getFormattedEnvArgComponent(cwd);
-  var runCommand = `run -w /app/src -p 3000:3000 --rm -it ${mountComponent} ${envArgComponent} ${imageId} sh -c "${command}" || true`;
+  var commandWithPort = command.replace("$PORT", 3000).replace("${PORT}", 3000)
+  var runCommand = `run -w /app/src -p 3000:3000 --rm -it ${mountComponent} ${envArgComponent} ${imageId} sh -c "${commandWithPort}" || true`;
```

Comment:
Does `/app/src` cause technical problems for compiled apps, or is the name just unintuitive? We could make it variable, though I'd love not to and I'm not tied to `/app/src`.

We need a directory that can be either mounted or copied, depending on situation (`exec` mounts, `start` and `release` copy). So all of the stuff that needs to hang around the container (like binary installs of `java`, `node`, etc) need a place to go `/app/heroku`. When the user's app dir is either mounted or copied, that needs a place to go that won't clobber the existing system.

Would `/app/lib` work better, or `/app/user`, or any other ideas?